src/test/java/de/tum/in/www1/artemis/TextAssessmentIntegrationTest.java

```diff
@@ -84,8 +84,7 @@ public void retrieveParticipationForSubmission_studentHidden() throws Exception
         TextSubmission textSubmission = ModelFactory.generateTextSubmission("Some text", Language.ENGLISH, false);
         textSubmission = database.addTextSubmission(textExercise, textSubmission, "student1");
 
-        StudentParticipation participationWithoutAssessment = request.get("/api/text-assessments/exercise/" + textExercise.getId() + "/submission/" + textSubmission.getId(),
-                HttpStatus.OK, StudentParticipation.class);
+        StudentParticipation participationWithoutAssessment = request.get("/api/text-assessments/submission/" + textSubmission.getId(), HttpStatus.OK, StudentParticipation.class);
```

Comment:
Prefer template literal over string concatenation.

Reply:
I agree in general. For consistency reasons with other test cases, I will leave it as it is right now.